### ISSUE TITLE
tomorrow pip package installs tests for no real reason

### ISSUE THREAD

**Issue by Paulius-Maruska** [madisonmay/Tomorrow#27]
Hello,

After doing `pip install tomorrow` - you will find that your tests directory is installed along with the tomorrow package into your site-packages.

Here are complete contents of `site-packages/tomorrow-0.2.3-py3.4.egg-info/installed-files.txt` file on my ubuntu box:

```
../tests/test.py
../tests/__init__.py
../tomorrow/__init__.py
../tomorrow/tomorrow.py
../tests/__pycache__/test.cpython-34.pyc
../tests/__pycache__/__init__.cpython-34.pyc
../tomorrow/__pycache__/__init__.cpython-34.pyc
../tomorrow/__pycache__/tomorrow.cpython-34.pyc
./
SOURCES.txt
dependency_links.txt
top_level.txt
requires.txt
PKG-INFO
```

Most packages do not distribute their own unit tests. The ones that do - have a uniquely named package, so that it would never clash with anything a user project might have.

This is how I found out about the bug, I was trying to `import tests.fixtures` in my own project, where file `tests/fixtures.py` exists and I kept getting an `ImportError`.

Please consider adding `exclude` argument in `find_packages` call in your `setup.py` file. Thanks.

**Comment by cx-lukas-salkauskas-x** [comment]
:+1: yes, `tests` does not belong in there...

**Comment by madisonmay** [comment]
Thanks for the recommendation, I'll definitely do that.  My apologies for the slow response time!

--Madison

**Comment by madisonmay** [comment]
Just pushed a fix with version 0.2.4.  Thanks again @Paulius-Maruska
